metricbeat/module/windows/perfmon/pdh_integration_windows_test.go

```diff
@@ -47,10 +47,8 @@ func TestQuery(t *testing.T) {
 
 func TestExistingCounter(t *testing.T) {
 	config := make([]CounterConfig, 1)
-	config[0].Name = "process"
-	config[0].Group = make([]CounterConfigGroup, 1)
-	config[0].Group[0].Alias = "processor_time"
-	config[0].Group[0].Query = processorTimeCounter
+	config[0].Alias = "process.processor_time"
```

Comment:
I know it just test code, but I would make this match what gets used in the example config (if you change it).